reana_workflow_controller/rest/utils.py

```diff
@@ -625,20 +606,33 @@ def get_workspace_diff(workflow_a, workflow_b, brief=False, context_lines=5):
             )
 
 
-def get_workflow_progress(workflow, include_progress=False):
+def get_most_recent_job_info(workflow_id: UUID) -> dict:
```

Comment:
`from typing import Dict`

```suggestion
def get_most_recent_job_info(workflow_id: UUID) -> Dict[str, str]:
```

[explanation why](https://stackoverflow.com/questions/39458193/using-list-tuple-etc-from-typing-vs-directly-referring-type-as-list-tuple-etc)